packages/imagemagick.rb

```diff
@@ -3,11 +3,11 @@
 class Imagemagick < Package
   description 'Use ImageMagick to create, edit, compose, or convert bitmap images.'
   homepage 'http://www.imagemagick.org/script/index.php'
-  version '7.0.6-4'
-  source_url 'https://www.imagemagick.org/download/releases/ImageMagick-7.0.6-4.tar.xz'
-  source_sha256 '5fe1ce7d78befb5c8aa7f8ae69710d0f78063d2e3a35c656521a3ce468ea733b'
+  version '7.0.6-7'
+  source_url 'https://www.imagemagick.org/download/releases/ImageMagick-7.0.6-7.tar.xz'
+  source_sha256 '732332a76cb62f067d680a90d85dd05a2f2592e0017af83becb639d05681106d'
```

Comment:
Why the big jump from -4 to -7?  It's an arbitrary version so it doesn't really matter.  I was just curious.